src/main/scala/stdlib/Traits.scala

```diff
@@ -11,7 +11,7 @@ import org.scalatest._
  */
 object Traits extends FlatSpec with Matchers with org.scalaexercises.definitions.Section {
 
-  /** Similar to interfaces in Java, traits are used to define object types by specifying the signature of the supported methods. Unlike Java, Scala allows traits to be partially implemented; i.e. it is possible to define default implementations for some methods. In contrast to classes, traits may not have constructor parameters.
+  /** Similar to interfaces in Java, traits are used to define object types by specifying the signature of the supported methods. And like in Java 8 does with interfaces and default methods in them, Scala allows traits to be partially implemented; i.e. it is possible to define default implementations for some methods. In contrast to classes, traits may not have constructor parameters.
```

Comment:
Hi! I think we don't necessarily have to compare to Java all the time. In the end it doesn't really help in conveying the message. What do you think about removing the part alltogether? It could make the paragraph simpler and easier to comprehend.

```suggestion
  /** Similar to interfaces in Java, traits are used to define object types by specifying the signature of the supported methods. In traits methods can have default implementations. In contrast to classes, traits may not have constructor parameters.
```

Reply:
I absolutely agree. Just wanted to make it less confusing for java developers, but no references to java are needed to understand traits haha